src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md

```diff
@@ -0,0 +1,77 @@
+---
+type: Blog
+title: 'Elastic Cloud Kubernetes: How to fix a corrupt index exception'
```

Comment:
Should this actually be "Elastic Cloud on Kubernetes"? Or is "Elastic Cloud Kubernetes" the thing?

https://www.elastic.co/guide/en/cloud-on-k8s/current/index.html

Reply:
If it is with "on" remember to update the image for this blog also.

Reply:
Hi, Its Elastic Cloud on Kubernetes